mappings/net/minecraft/data/server/BiomeParameterProvider.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/unmapped/C_wwetinhi net/minecraft/data/server/BiomeParameterProvider
```

Comment:
Looking at Mojmap's name (and therefore the dev's intention) and double-checking the code, this should instead be `BiomeParameterDumpProvider`, since it's dumping at the `reports` directory